source/type/type.h

```diff
@@ -160,23 +156,23 @@ struct StopPoint : public Header, Nameable, hasProperties, HasMessages {
     std::vector<StopPointConnection*> stop_point_connection_list;
     std::set<Dataset*> dataset_list;
 
-    template<class Archive> void serialize(Archive & ar, const unsigned int ) {
+    template <class Archive>
+    void serialize(Archive& ar, const unsigned int) {
         // The *_list are not serialized here to avoid stack abuse
         // during serialization and deserialization.
         //
         // stop_point_connection_list is managed by StopPointConnection
-        ar & uri & label & name & stop_area & coord & fare_zone & is_zonal & idx & platform_code
-            & admin_list & _properties & impacts & dataset_list;
+        ar& uri& label& name& stop_area& coord& fare_zone& is_zonal& idx& platform_code& admin_list& _properties&
```

Comment:
the spacing was nice here :(

Reply:
Unfortunately, I couldn't find any configuration to fix this :(